test.sh

```diff
@@ -1,4 +1,5 @@
 npm install
+node seed.js
 npm run test-mocha
 npm start &
 # node seed.js
```

Comment:
`npm run test-mocha` already runs the tests, which ideally will have as first tests the ones for the models, which test adding data to the db.
So `node seed.js` should later be done where this commented line is, similarly to how it was done in the original example; it drops the whole database and reseeds it for the `nightwatch` tests.